pkg/features/versioned_kube_features.go

```diff
@@ -29,7 +27,8 @@ import (
 // Entries are separated from each other with blank lines to avoid sweeping gofmt changes
 // when adding or removing one entry.
 var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate.VersionedSpecs{
-	genericfeatures.EmulationVersion: {
-		{Version: version.MustParse("1.30"), Default: false, PreRelease: featuregate.Alpha},
-	},
+	// Example:
+	// genericfeatures.EmulationVersion: {
+	// 	{Version: version.MustParse("1.30"), Default: false, PreRelease: featuregate.Alpha},
+	// },
```

Comment:
needed?

Reply:
Yes, this is just to give an example, because we don't have any versioned feature right now.